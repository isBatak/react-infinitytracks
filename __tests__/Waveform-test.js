import React from 'react';
import { Waveform } from 'react-infinitytracks';
import renderer from 'react-test-renderer';

const audioCtx = new AudioContext();
const myArrayBuffer = audioCtx.createBuffer(2, audioCtx.sampleRate * 3, audioCtx.sampleRate);

test('Show wavefrom', () => {
  const component = renderer.create(
    <Waveform buffer={myArrayBuffer} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
