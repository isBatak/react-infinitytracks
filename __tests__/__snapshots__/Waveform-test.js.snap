// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Show waveform 1`] = `
<canvas
  height={100}
  width={720}
/>
`;
